pkg/setting/setting_openfeature.go

```diff
@@ -0,0 +1,51 @@
+package setting
+
+import (
+	"fmt"
+	"net/url"
+)
+
+const (
+	StaticProviderType = "static"
+	GOFFProviderType   = "goff"
+)
+
+type OpenFeatureSettings struct {
+	ProviderType string
+	URL          *url.URL
+	TargetingKey string
+	ContextAttrs map[string]any
+}
+
+func (cfg *Cfg) readOpenFeatureSettings() error {
+	cfg.OpenFeature = OpenFeatureSettings{}
+
+	config := cfg.Raw.Section("feature_toggles.openfeature")
+	cfg.OpenFeature.ProviderType = config.Key("provider").MustString(StaticProviderType)
+	cfg.OpenFeature.TargetingKey = config.Key("targetingKey").MustString(cfg.AppURL)
+
+	strURL := config.Key("url").MustString("")
+
+	if strURL != "" && cfg.OpenFeature.ProviderType == GOFFProviderType {
+		u, err := url.Parse(strURL)
```

Comment:
Unfortunately `url.Parse("")` won't error, so we also need to check for that:

```suggestion
		if strURL == "" {
			return fmt.Errorf("must provide a valid URL with goff feature provider")
		}
		u, err := url.Parse(strURL)
```